bin/remixd

```diff
@@ -1,46 +1,37 @@
 #!/usr/bin/env node
-var Router = require('../src/router')
-var servicesList = require('../src/servicesList')
-var program = require('commander')
+let origins = require('../public/origins.json');
```

Comment:
You're not reassigning this value (just mutating it). So you can use `const`.

Reply:
Updated and resolving